RadixWallet/Profile/Entity/Account.swift

```diff
@@ -143,6 +143,13 @@ extension Profile.Network.Account {
 			control.transactionSigning.factorSourceID.kind == .ledgerHQHardwareWallet
 		}
 	}
+
+	public var deviceFactorSourceID: FactorSourceID.FromHash {
```

Comment:
We cannot have this... `deviceFactorSourceID`... MUST be optional, and you MUST check that the factor source kind of `control.transasctionSigning` is indeed `.device`. It could be `.ledger`!